packages/core/src/model.ts

```diff
@@ -180,6 +180,11 @@ export class ChartModel {
 			});
 		}
 
+		// dispatch legend filtering event with the status of all the dataLabels
+		this.services.events.dispatchEvent("legend-items-update", {
```

Comment:
We should use an ENUM for the event name like so https://github.com/carbon-design-system/carbon-charts/blob/79c1bc714aa9c6bd6971c49cf92c4122e7faf84e/packages/core/src/components/graphs/bar-simple.ts#L142